gcd/src/main/java/org/spine3/server/storage/datastore/DatastoreWrapper.java

```diff
@@ -318,4 +365,40 @@ private KeyFactory initKeyFactory(String kind) {
         keyFactories.put(kind, keyFactory);
         return keyFactory;
     }
+
+    private List<Entity> readBigBulk(List<Key> keys) {
```

Comment:
Let's have this method only and rename it to `readBulk`.
`smallBulk` is irrelevant, since it is impossible to guess, if the data is huge.